eng/Publishing.props

```diff
@@ -95,7 +95,7 @@
   </Target>
 
   <!-- We use a separate target to publish this to blob storage so that we can push this to a relative path inside the blob storage. -->
-  <Target Name="PublishToolsetAssets" DependsOnTargets="ReadToolsetVersion" BeforeTargets="Publish">
+  <Target Name="PublishToolsetAssets" DependsOnTargets="ReadToolsetVersion" BeforeTargets="Publish" Condition="'$(EnableToolsetPublish)' == 'true'">
```

Comment:
`EnableToolsetPublish` was added since it should only run once per pipeline run. So, this is only specified for the main Windows job.

Reply:
Can this be conditioned on `EnableDefaultArtifacts` eq true instead?